packages/material/src/layouts/MaterialArrayLayout.tsx

```diff
@@ -148,25 +163,58 @@ class ExpandPanelRenderer extends React.Component<ExpandPanelProps, any> {
       <ExpansionPanel expanded={expanded} onChange={handleExpansion(childPath)}>
         <ExpansionPanelSummary expandIcon={<ExpandMoreIcon />}>
           <Grid container alignItems={'center'}>
-            <Grid item xs={11}>
+            <Grid item xs={10}>
               <Grid container alignItems={'center'}>
                 <Grid item xs={1}>
-                  <Avatar aria-label='Index'>{index + 1}</Avatar>
+                  <Avatar aria-label="Index">{index + 1}</Avatar>
                 </Grid>
                 <Grid item xs={2}>
                   {childLabel}
                 </Grid>
               </Grid>
             </Grid>
-            <Grid item xs={1}>
+            <Grid item xs={2}>
               <Grid container justify={'flex-end'}>
                 <Grid item>
-                  <IconButton
-                    onClick={removeItems(path, [index])}
-                    style={iconStyle}
+                  <Grid
+                    container
+                    direction="row"
+                    justify="center"
+                    alignItems="center"
                   >
-                    <DeleteIcon />
-                  </IconButton>
+                    {uischema.options && uischema.options.showSortButtons ? (
```

Comment:
@eneufeld `why do we check the up/down buttons individually? I think it would be easier to simply reuse the props from the table`
Can you elaborate this more. I am not able to follow over here.

Reply:
You can ignore it now, in the first version you had the `uischema.options && uischema.options.showSortButtons` check before each button. Now it is fine!